scrapi/processing/cassandra.py

```diff
@@ -32,7 +32,7 @@ def process_normalized(self, raw_doc, normalized):
             contributors=copy_to_unicode(json.dumps(normalized['contributors'])),
             description=copy_to_unicode(normalized.get('description')),
             uris=copy_to_unicode(json.dumps(normalized['uris'])),
-            providerUpdatedDateTime=parse(normalized['providerUpdatedDateTime']),
+            providerUpdatedDateTime=parse(normalized['providerUpdatedDateTime']).replace(tzinfo=None),
```

Comment:
Why strip the tzinfo here, where as in the date_formatter helper you're adding tzinfo? Is this a Cassandra thing?